interface/forms/eye_mag/help.php

```diff
@@ -455,520 +615,1032 @@
-                                    <td><strong class="text-danger">dantseg;</strong><br /><strong class="text-danger">DAS;</strong></td>
-                                    <td>All Anterior Segment fields with defined default values are <strong>erased</strong> and filled with default values.<br />Anterior Segment Fields without defined default values and all other fields on the form are not affected. </td>
-                                </tr>
-                                <tr >
-                                    <td>Conjunctiva</td><td>Right = rc<br />Left = lc<br />Both = bc or c</td>
-                                    <td><span class="field">rc:</span>+1 inj<br /><span class="field">c:</span>med pter</td>
-                                    <td>"+1 injection" (right conj only)<br />"medial pterygium" (both right and left fields are filled)</td>
-                                </tr>
-                                <tr>
-                                    <td>Cornea</td><td>Right = rc<br />Left = lc<br />Both = bk or k</td>
-                                    <td><span class="field">rk:</span>+3 spk<br /><span class="field">k:</span>+2 end gut<strong class="text-success">;</strong><span class="field">rk:</span>+1 str edema<strong class="text-success">.a</strong></td>
-                                    <td>"+3 SPK" (right cornea only)<br />"+2 endothelial guttatae" (both cornea fields) AND "+1 stromal edema" (appended to Right cornea field)</td>
-                                </tr>
-                                <tr>
-                                    <td>Anterior Chamber</td><td>Right = rac<br />Left = lac<br />Both = bac or ac</td>
-                                    <td><span class="field">rac:</span>+1 fc<br /><span class="field">ac:</span>+2 flare</td>
-                                    <td>"+1 flare/cell" (right A/C field only)<br />"+2 flare" (both A/C fields)</td>
-                                </tr>
-                                <tr>
-                                    <td>Lens</td><td>Right = rl<br />Left = ll<br />Both = bl or l</td>
-                                    <td><span class="field">RL:</span>+2 NS<br /><span class="field">ll:</span>+2 NS<strong class="text-success">;</strong><span class="field">l:</span>+3 ant cort spokes.a</td>
-                                    <td>"+2 NS" (right lens only)<br />"+2 NS" (left lens fields) AND "+3 anterior cortical spokes" (appended to both lenses)</td>
-                                </tr>
-                                <tr>
-                                    <td>Iris</td><td>Right = ri<br />Left = li<br />Both = bi or i</td>
-                                    <td><strong class="text-danger">bi.</strong>12 0 iridotomy<br /><span class="field">ri:</span>+2 TI defects<strong class="text-success">.a</strong><strong style="color:navy">;</strong><span class="field">li</span>.round</td>
```

Comment:
Also, I found here has inline-style color, So I removed it, please take a look. Is it necessary?